packages/office-ui-fabric-react/src/components/ScrollablePane/ScrollablePane.base.tsx

```diff
@@ -191,14 +191,16 @@ export class ScrollablePaneBase extends BaseComponent<IScrollablePaneProps, {}>
 
   private _resizeContainer() {
     const { stickyContainer, root } = this.refs;
-    const { borderTopWidth, borderLeftWidth } = getComputedStyle(root);
-    stickyContainer.style.height = root.clientHeight + 'px';
-    stickyContainer.style.width = root.clientWidth + 'px';
-    if (borderTopWidth) {
-      stickyContainer.style.top = root.offsetTop + parseInt(borderTopWidth, 10) + 'px';
-    }
-    if (borderLeftWidth) {
-      stickyContainer.style.left = root.offsetLeft + parseInt(borderLeftWidth, 10) + 'px';
+    if (root && stickyContainer) {
```

Comment:
This is hiding the root cause. Why is the function being called when refs are null? Because there is a setTimeout calling it after an unmount.

Reply:
Ah. didnt see this till after I submitted the clearTimeout changes. Is this better than manually calling clearTimeout?

Reply:
This._async.setTimeout worked perfect